pcsx2/gui/AppConfig.cpp

```diff
@@ -883,6 +884,7 @@ void AppConfig::GSWindowOptions::LoadSave( IniInterface& ini )
 	IniEntry( IsFullscreen );
 
     IniEntry( IsToggleFullscreenOnDoubleClick );
+	IniEntry(IsToggleAspectRatioSwitch);
```

Comment:
You might as well fix it (and also, add spaces around inientry arg,to be more consistent with surrounding style).
pcsx2 coding style is something nonexistent.